base/markdown/parse/util.jl

```diff
@@ -93,7 +93,7 @@ function startswith{T<:String}(stream::IO, ss::Vector{T}; kws...)
 end
 
 function startswith(stream::IO, r::Regex; eat = true, padding = false)
-  @assert beginswith(r.pattern, "^")
+  @assert Base.startswith(r.pattern, "^")
```

Comment:
Yup – I'm fine with a rename if we can come up with one, but since this is self-contained (and isn't adding methods to `Base.startswith`) it doesn't seem like too much of a problem.